package.json

```diff
@@ -12,14 +12,17 @@
   "license": "ISC",
   "devDependencies": {
     "@types/express": "^4.17.21",
-    "@types/node": "^20.14.0",
     "nodemon": "^3.1.2",
     "ts-node": "^10.9.2",
     "typescript": "^5.4.5"
   },
   "dependencies": {
+    "dotenv": "^16.4.5",
     "express": "^4.19.2",
     "mongodb": "^6.7.0",
-    "mongoose": "^8.4.1"
+    "mongoose": "^8.4.1",
+    "nconf": "^0.12.1",
+    "passport": "^0.7.0",
```

Comment:
Passport is not used in this PR, maybe we can move it to the PR where it is used.